### PR TITLE
update valid cnzz url

### PR DESCRIPTION
**Pull request by lcy362**
The current cnzz stat url is `https://s13.cnzz.com/z_stat.php?`  ， update that

### CODE REVIEW

File: layout/_third-party/analytics/cnzz-analytics.swig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% if theme.cnzz_siteid %}
 
   <div style="display: none;">
-    <script src="//s95.cnzz.com/z_stat.php?id={{ theme.cnzz_siteid }}&web_id={{ theme.cnzz_siteid }}" language="JavaScript"></script>
+    <script src="https://s13.cnzz.com/z_stat.php?id={{ theme.cnzz_siteid }}&web_id={{ theme.cnzz_siteid }}" language="JavaScript"></script>
```
[the remark this answers is not in the document]
Comment (lcy362) [reply layout/_third-party/analytics/cnzz-analytics.swig]: Yeah, that's correct. Updated.

**Comment by geekrainy** [comment]
I think both addresses are available.

**Comment by sli1989** [comment]
Is there any reference links about the change `s13`?

**Comment by sli1989** [comment]
oh, closed? anyway, thanks for your time.

**Pull request closed without merging**